CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+## 4.16
+
+* Update Go analyzer 1.18 -> [1.19](https://sonarsource.atlassian.net/issues/?jql=fixVersion%20=%2016206%20ORDER%20BY%20created%20ASC)
+* Update PHP analyzer 3.42 -> [3.43](https://sonarsource.atlassian.net/projects/SONARPHP/versions/16340/tab/release-report-all-issues)
```

Comment:
```suggestion
* Update PHP analyzer 3.42 -> [3.43](https://sonarsource.atlassian.net/issues/?jql=fixVersion%20%3D%2016340%20ORDER%20BY%20created%20ASC)
```
Unfortunately, lots of analysis teams provide us with wrong links that are not publicly accessible. The only publicly accessible URLs are from the `issue navigator`.